CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+Welcome and thanks that you want to contribute to the Trustlines Project.
+We appreciate all help, depending on your knowledge, this can be
+spreading the word, helping with documentation, reporting issues or even contributing to the source code.
+
+Feel free to checkout the code, play around with it and run the tests.
+Before making a change, please first discuss the change via opening an issue on this
+repository. If you just need some small question answered, you can also reach out to
+us via our [gitter chat](https://gitter.im/trustlines/community).
+
+## Open an issue
+
+To request a feature or report a bug, please open an issue on the issue tracker in github.
+
+### Report a bug
+
+Please include a short description of the problem, including what you expected to happen and what did happen instead.
+Please also include the version of the relay you were using, your operating system and the used configuration.
+
+### Request a feature
+
+Please include a description of what you would like to see implemented and an explanation of why you believe this would
+ be a good addition.
+
+## Open a pull request
+
+### Before starting
+- Follow the [readme](/README.rst) to setup you dev environment.
+- Ensure that you can run the tests and that they pass. The tests can be run from the root with
+`./pytest tests`
+- Ensure that you can run the end2end tests. You can find them in the
+[end2end repo](https://github.com/trustlines-protocol/end2end):
+  First build the docker image locally from the root of this repo with
+  `docker build -t <tag-name> .`
+  Then follow the instructions of the end2end repository to run the tests with that image.
+
+### To include
+
+- For a bug fix:
+  - Add a test that makes the bug explicit, and make sure that the test fails
+  - Fix the bug, so that the test passes
+- For a new feature:
+  - Add a test, that demonstrate the new feature. Consider also adding the new functionality
+  together with an end2end test to the
```

Comment:
I think here we should also mention the relay.

Reply:
You want to say something like: `Implement the corresponding changes in the relay, or open an issue for it`?

Reply:
yes, or remove it all together. I think it is weird only to mention the clientlib

Reply:
I removed it.